### PR TITLE
dashboard: reposition closed meeting icon

### LINKED ISSUE #3987: Closed meeting: Icon in calendar listing needs repositioning
**Description**
The lock icon, which indicates a closed meeting, should be repositioned. The current position is not optimal. 

Current view:
image

**How to Reproduce**
Requirement: A closed meeting is needed, which could be seen in the calender overview.

**Expected behavior**
The lock icon should be placed in the meeting name line right before the meeting name. The size of the icon should fit the font size of the meeting name.

### PR DESCRIPTION
**Pull request by reiterl**
Resolve #3987 

Use the os-icon-container part to display icon and meeting name.

### CODE REVIEW

**Review by rrenkert — COMMENTED** [review]
Fits the requirements described in the issue. Might look better if the meeting date is aligned to the name and not the icon.
@emanuelschuetze  what do you think? And please also have a look at the meeting list on orga level.

**Comment by rrenkert** [comment]
Talked to @emanuelschuetze: We want to have the date aligned to the name. Please also fix the alignment according to the calendar in the meeting list on orga level.

File: client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html [hunk 1]
```diff
@@ -78,7 +78,17 @@ <h2>{{ 'Meetings' | translate }}</h2>
...
+                        class="subtitle ellipsis-overflow"
+                        [ngClass]="{
+                            'align-w-title-w-one-icon': xor(
+                                xor(meeting.isTemplate, meeting.locked_from_inside),
+                                meeting.isArchived
+                            ),
+                            'align-w-title-w-two-icon':
+                                meeting.isArchived && (meeting.isTemplate || meeting.locked_from_inside)
+                        }"
+                    >
```
Comment (bastianjoel) [inline client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html:91]:
```suggestion
                    @let numIcons = meeting.isArchived + meeting.isTemplate + meeting.locked_from_inside
                    <div
                        class="subtitle ellipsis-overflow"
                        [ngClass]="{
                            'align-w-title-w-one-icon': numIcons === 1,
                            'align-w-title-w-two-icon': numIcons >= 2
                        }"
                    >
```

And remove `xor` from `meeting-list.component.ts`

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html (hunk 1 above)
Comment (Elblinator) [reply client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html]: meeting.locked_from_inside can be undefined which results in numIcons to be NaN

**Review by rrenkert — APPROVED** [review]

File: client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html [hunk 2]
```diff
@@ -78,7 +78,17 @@ <h2>{{ 'Meetings' | translate }}</h2>
                 </div>
 
                 @if (meeting.location || meeting.start_time || meeting.end_time) {
-                    <div class="subtitle ellipsis-overflow">
+                    @let numIcons =
+                        meeting.isArchived +
+                        meeting.isTemplate +
+                        (meeting.locked_from_inside ? meeting.locked_from_inside : false);
```
Comment (bastianjoel) [inline client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html:84]:
```suggestion
                    @let numIcons = meeting.isArchived + meeting.isTemplate + !!meeting.locked_from_inside;
```

**Review by bastianjoel — CHANGES_REQUESTED** [review]

File: client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html (hunk 2 above)
Comment (Elblinator) [reply client/src/app/site/pages/organization/pages/orga-meetings/pages/meeting-list/components/meeting-list/meeting-list.component.html]: done :)

**Review by bastianjoel — APPROVED** [review]

**Pull request merged**